aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -0,0 +1,195 @@
+
+MVC benefits, in addition to the server rendering benefits previously listed:
+
+* Easier to scale the web UI application in terms of complexity. It's easier to code, debug, and test something (model, view, or controller) that has a single job.
+* Clear [separation of concerns](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#separation-of-concerns) for maximum flexibility.
+* The Model-View-Controller separation of responsibilities ensures that the business model can evolve without being tightly coupled to low-level implementation details.
+
+To get started with your first ASP.NET Core MVC app, see [Get started with MVC](xref:tutorials/first-mvc-app/start-mvc). For a complete overview of ASP.NET Core MVC, its architecture and benefits, see [Overview of ASP.NET Core MVC](xref:mvc/overview)
+
+## Choose a client rendered ASP.NET Core solution
+
+The following section briefly explains the ASP.NET Core web UI client rendered models available and directs you to the next steps to get started.
+
+### ASP.NET Core Blazor WebAssembly
+
+Blazor WebAssembly is a [single-page app (SPA) framework](/dotnet/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps) for building interactive client-side web apps with .NET. Blazor WebAssembly uses open web standards without plugins or recompiling code into other languages. Blazor WebAssembly works in all modern web browsers, including mobile browsers.
+
+Running .NET code inside web browsers is made possible by [WebAssembly](https://webassembly.org) (abbreviated `wasm`). WebAssembly is a compact bytecode format optimized for fast download and maximum execution speed. WebAssembly is an open web standard and supported in web browsers without plugins.
+
+WebAssembly code can access the full functionality of the browser via JavaScript, called *JavaScript interoperability*. .NET code executed via WebAssembly in the browser runs in the browser's JavaScript sandbox with the protections that the sandbox provides against malicious actions on the client machine.
+
+Benefits:
+
+* Build interactive web UIs using C# instead of JavaScript. Using the same language for front-end and back-end code can:
+  * Accelerate app development.
+  * Reduce build pipeline complexity.
+  * Simplify maintenance.
+  * Let developers understand and work on both client-side and server-side code.
+* Reusable, sharable UI components: Blazor apps are composed of reusable web UI components implemented using C#, HTML, and CSS. Both client and server code  are written in C#, allowing you to share code and libraries.
+* Blazor works in all modern web browsers, including mobile browsers. Blazor uses open web standards without plug-ins or code transpilation.
+* Quickly get productive with Blazor reusable UI components from top component vendors like:
```

Comment:
This list of component vendors has to get updated regularly. We have a list that we maintain on the https://blazor.net page. I'd prefer not to have to maintain a parallel list here.